features-json/gamepad.json

```diff
@@ -219,10 +219,27 @@
       "30":"n"
     },
     "and_chr":{
-      "44":"n"
+      "36":"n",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"y",
+      "41":"y",
+      "42":"y",
+      "43":"y",
```

Comment:
@luser It's not really super-clear, but it is mentioned in https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md (you can't add new versions, can only modify values). So yes, only the latest version for mobile Chrome & Firefox right now. I'll update CONTRIBUTING.md to make that more clear.